ui/projekt-monsterz-front/README.md

```diff
@@ -0,0 +1,27 @@
+# ProjektMonsterzFront
```

Comment:
You could keep the README in here, but it doesn't really serve as much of a help. Might be worth removing?